spec/models/library_entry_spec.rb

```diff
@@ -134,6 +135,49 @@
     end
   end
 
+  describe 'timestamp validation' do
+    let!(:anime) { create(:anime) }
+    let!(:library_entry) do
+      create(:library_entry, media: Anime.last, status: :current, progress: 1)
```

Comment:
read above. Also don't reference Anime.last here if it is in the same block. You can just do `media: :anime` because the variable is in the same scope. You would do `Anime.last` if this was in a `before(:each)` or some other block and you didn't have access to the variable, then you would find the entry it created in the database.